### DISCUSSION TITLE
make ratio as float not integer.

### DISCUSSION THREAD

**Discussion by LinArcX** [sxyazi/yazi#593] [4. Ideas]
Currently the only numbers that we can specify for ratio are integers as i understood:
```
[manager]
ratio          = [ 2, 4, 2 ]
```

Can we have float numbers instead for better flexibility?

**Comment by sxyazi** [comment]
You can multiply it by 10 or something, like `ratio = [ 20, 40, 20 ]`, which has better flexibility
